.bcr/presubmit.yml

```diff
@@ -0,0 +1,12 @@
+bcr_test_module:
+  module_path: "examples"
+  matrix:
+    platform: ["debian10", "macos", "ubuntu2004", "windows"]
+    bazel: ["7.x", "6.x"]
+  tasks:
+    run_tests:
```

Comment:
Should we add a second job for testing the examples?

Reply:
We *only* test the examples, see line 2.

There are a few options for how BCR does testing, IMO the value is from testing the module from external as a user, and that's how we've used this on most rulesets I work on. Can discuss more